sbin/build.sh

```diff
@@ -132,7 +132,7 @@ configureReproducibleBuildParameter() {
 
       # Ensure reproducible and comparable binary with a unique build user identifier
       addConfigureArg "--with-build-user=" "admin"
-      if [ "${BUILD_CONFIG[OS_KERNEL_NAME]}" == "aix" ]; then
+      if [ "${BUILD_CONFIG[OS_KERNEL_NAME]}"  == "aix" ] && [ "${JDK_VERSION_NUMBER}" -lt 22 ]; then
```

Comment:
From the builds logs, it looks like JDK_VERSION_NUMBER isn't available here

Reply:
How does the build pass for JDK8 then ?

Reply:
I am facing this error too now, after removing extra brackets. I am using the OPENJDK map to access it now